### DISCUSSION TITLE
Modify PnP People Template as provided by SharePoint Online Search

### DISCUSSION THREAD

**Discussion by vishalgoyal2472** [microsoft-search/pnp-modern-search#4092] [General]
We want to show people layout in two columns,

1.  In first column- Name, Designation and Department 
2.  In second column Email, chat option and Phone number?

 

<img width="512" alt="image" src="https://github.com/user-attachments/assets/d59f4a35-4e60-4e13-8251-3fe8e184232e">

**Comment by kasperbolarsen** [comment]
Sure, you just have to develop you own display layout, it is just HTML and CSS after all. You might be able to find inspiration at https://github.com/microsoft-search/pnp-modern-search-layouts

**Reply by vishalgoyal2472** [reply]
thank you for your response but which property shows Department, Office Location and user team link,

I think team link we need to manage through anchor tag.

I am using BaseOfficeLocation property but its showing null.

**Reply by kasperbolarsen** [reply]
Using one of the tools here, https://microsoft-search.github.io/pnp-modern-search/using-query-tools-to-verify-issues/ combined with looking at the values in the User Profile Application is usually a good starting point
